### DISCUSSION TITLE
【求助】兄弟们救一下！！！

### DISCUSSION THREAD

**Discussion by yesui4262** [share121/inter-knot#3401] [其他]
![run](https://github.com/user-attachments/assets/e67bd941-7936-4d46-9ccb-20dd39b48c6e)
一下地铁就被一个大屁股单马尾和一个小个子双马尾警官追，这怎么回事？！！
有没有在附近的兄弟救一下！！

**Comment by HyperMuteki0347** [comment]
建议乖乖站好等着被带走就好了，可以少吃点苦头，否则鼻青脸肿地进局子。（PS：听说治安局监狱犯人的伙食和待遇挺好，不会受委屈。还有你犯了啥事为啥要跑？）<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Wax1234567** [comment]
看装扮，那两位不像什么善茬，不过您还有功夫发绳网，也不是什么等闲之辈啊<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >
记得别出卖绳网的兄弟们<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
